src/main/kotlin/app/CodeLongevity.kt

```diff
@@ -0,0 +1,173 @@
+// Copyright 2017 Sourcerer Inc. All Rights Reserved.
+
+package app
+
+import org.eclipse.jgit.blame.BlameGenerator
+import org.eclipse.jgit.blame.BlameResult
+import org.eclipse.jgit.diff.DiffFormatter
+import org.eclipse.jgit.diff.DiffEntry
+import org.eclipse.jgit.diff.RawText
+import org.eclipse.jgit.internal.storage.file.FileRepository
+import org.eclipse.jgit.lib.Repository
+import org.eclipse.jgit.revwalk.RevCommit
+import org.eclipse.jgit.revwalk.RevWalk
+import org.eclipse.jgit.revwalk.RevSort
+import org.eclipse.jgit.treewalk.filter.*
+import org.eclipse.jgit.treewalk.TreeWalk
+import org.eclipse.jgit.util.io.DisabledOutputStream
+
+import java.util.Date
+
+/**
+ * Used to store and identify a code line information in a repo.
+ */
+class CodeLine(val rev: String, val file: String, val line: Int,
```

Comment:
repo is a common slang. rev is not. Compare: https://en.wikipedia.org/wiki/Rev vs https://en.wikipedia.org/wiki/Repo_(disambiguation)